Lib/fontTools/ttLib/tables/_p_o_s_t.py

```diff
@@ -252,14 +259,17 @@ def fromXML(self, name, attrs, content, ttFont):
 			self.data = readHex(content)
 
 
-def unpackPStrings(data):
+def unpackPStrings(data, nbStrings):
 	strings = []
 	index = 0
-	dataLen = len(data)
-	while index < dataLen:
+	while nbStrings > 0:
 		length = byteord(data[index])
 		strings.append(tostr(data[index+1:index+1+length], encoding="latin1"))
 		index = index + 1 + length
+		nbStrings = nbStrings - 1
```

Comment:
```suggestion
		nbStrings -= 1
```

Reply:
Does "nb" in `nbStrings` stand for "NumBer" of strings? I think just `n` or `num` would be clear, as the function name gives context about the thing we are counting with that number.

Reply:
apologies for the bikeshedding; thinking some more about nbStrings parameter, maybe `maxStrings` reads better, as it's the _maximum_ number of ps strings we want to be read from the string data.

Reply:
In addition to extra bytes, is it ever possible that we don't have enough bytes, that is maxStrings is greater than the number of strings actually encoded in the string data? if so, do we get an IndexError at the next iteration following the one where we run out of data?

Reply:
Yes, not enough bytes is a possibility. Should that generate a log.warning (and use "" for missing strings) or a log.error?